test/usm/pool.hpp

```diff
@@ -0,0 +1,48 @@
+/*
+ *
+ * Copyright (C) 2023 Intel Corporation
+ *
+ * Part of the Unified-Runtime Project, under the Apache License v2.0 with LLVM Exceptions.
+ * See LICENSE.TXT
+ * SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+ *
+ */
+
+#ifndef USM_TEST_POOL_HPP
+#define USM_TEST_POOL_HPP 1
+
+#include <gtest/gtest.h>
+
+#include <umf/base.h>
+#include <umf/memory_provider.h>
+
+#include "umf_helpers.hpp"
+
+namespace usm_test {
+
+struct null_pool {
```

Comment:
I think we have null pool implementation already, as well.

Reply:
https://github.com/oneapi-src/unified-runtime/blob/main/test/unified_malloc_framework/common/pool.h#L15